src/Domain/Creators/BinaryCompactObject.cpp

```diff
@@ -415,8 +417,8 @@ Domain<3> BinaryCompactObject::create_domain() const {
             std::back_inserter(maps));
 
   Maps maps_first_outer_shell = sph_wedge_coordinate_maps<Frame::Inertial>(
-      radius_enveloping_cube_, radius_enveloping_sphere_, 0.0, 1.0,
-      use_equiangular_map_, 0.0, true, 1.0, 2, {},
+      radius_enveloping_cube_, radius_enveloping_sphere_, frustum_sphericity_,
+      1.0, use_equiangular_map_, 0.0, true, 1.0, 2, {},
```

Comment:
I accidentally pushed a commit to this branch where I tried something, sorry for that! Not sure how it's even possible that I can push to a branch on your fork. Please feel free to just delete the commit.

I suggest here to elide the "first outer shell" entirely when it isn't needed anymore, i.e. when `frustum_sphericity == 1.` _and_ the user hasn't explicitly specified an `OuterShell.InnerRadius`. The commit I accidentally pushed does that, so feel free to take code from it if you want. I or you can also do this in a follow-up prefer if you prefer.

Reply:
Unless the creator of a PR unchecks a box, maintainers can push to a PR.